### ISSUE TITLE
Permanently deactivate Auto Play next Video?

### ISSUE THREAD

**Issue by Thirteen-37** [yuliskov/SmartTube#2390]
Since one of the last updates, the app no longer remembers what it is supposed to do at the end of the video.
Normally, I had it set to stop after the end of the video and not to play the next video.
Now, every time the app is started, the videos are no longer stopped at the end, but auto-play is active.
Is there a way to change this in the settings, or is this a bug in the software?

**Comment by jg-ee** [comment]
Agree it does not remember to not autoplay anymore. Would like to it to remember or always default to do not autoplay.

**Comment by oneseventhree** [comment]
Agreed, It should be an option in 'Settings' for permanently disable auto play and not in when playing the video. It's the ONLY bad thing about this app. other than that its perfect!

**Comment by asandhu3** [comment]
Seems like a new bug, this was working as expected prior to latest update I believe.

**Comment by kajer** [comment]
This happened in the last few updates.  the last few updates change the play next setting back to auto.  This was never the case with the hundreds of updates (beta) until recently.  I made sure to check the setting with the last beta update, and indeed the play next setting was enabled after updating.

Very annoying to lose that setting after it being persistent for years.

**Comment by swagbbq** [comment]
same issue with 17.61 on shield tv

**Comment by mekarawr** [comment]
same issue on android tv since a couple of updates ago (beta)

**Comment by Skrapaa** [comment]
Same here

**Comment by waccii** [comment]
Same problem. 

It's important to me because my little brother keeps turning off the TV instead of the "Mi stick" , one time I turned on the TV, the youtube kept playing

That's why I disable & hide that function from the settings

**Comment by aecarvalho** [comment]
It can be done by directly playing a video, pausing it then pressing on the 🔁(pause/stop playback after each video) button next to the ⏭️ (next vide

**Comment by AbuD3vTs4113** [comment]
Fixed with 17.62 ✅

**Comment by decha00adebug** [comment]
Issue has seemingly returned with the latest beta update. I have tried changing the playback in settings and on the video player ui but neither works.
